jenkins/scripts/dynamic_worker_workflow/feature_tests.sh

```diff
@@ -68,6 +68,10 @@ if [[ "${UPDATED_REPO}" != *"${REPO_ORG}/${REPO_NAME}"* ]] ||
     git config user.name "Test"
     git remote add test "${UPDATED_REPO}"
     git fetch test
+    # if triggered from prow we cannot get the ghprbAuthorRepoGitUrl then we pull the PR
+    if [[ -n "$PR_ID" ]]; then
```

Comment:
```suggestion
    if [[ -n "${PR_ID:-}" ]]; then
```
Otherwise it'll fail to `set -u` being unset.

Reply:
done